src/algorithm/distance.rs

```diff
@@ -151,6 +164,32 @@ impl<T> Distance<T, Polygon<T>> for Point<T>
     }
 }
 
+// Minimum distance from a Point to a MultiPolygon
+impl<T> Distance<T, MultiPolygon<T>> for Point<T>
+    where T: Float
+{
+    fn distance(&self, mpolygon: &MultiPolygon<T>) -> T {
```

Comment:
I think if you did:

```rust
    /// Minimum distance from a Point to a MultiPolygon
    fn distance(&self, mpolygon: &MultiPolygon<T>) -> T {
```

It would show up in documentation (via rustdoc), just to be explicit about the behavior here.